ocean_provider/requests_session.py

```diff
@@ -13,7 +13,7 @@ def get_requests_session() -> Session:
     :return: requests session
     """
     session = Session()
-    retries = Retry(total=5, backoff_factor=1, status_forcelist=[502, 503, 504])
```

Comment:
how about these ones?

Reply:
we should keep retries in case of anything.